### ISSUE TITLE
PDFs from OSA failing to download on non-open-access articles

### ISSUE THREAD

**Issue by miromarszal** [zotero/translators#1568]
I noticed that Zotero occasionally fails to download PDFs from OSA. After a while I realized that this always happens to restricted-access articles. Open-access are fine. Is Zotero boycotting the paywall? ;)

Here's an example:
https://www.osapublishing.org/josab/abstract.cfm?uri=josab-11-1-231

**Comment by AbeJellinek** [comment]
No, unfortunately the site is boycotting Zotero! The `citation_pdf_url` meta tag should go straight to the PDF, but it instead goes to a viewer page that embeds it. Given the site's captcha protection, I don't know that there's any way for us to reliably fetch the PDF URL from that page.

**Comment by miromarszal** [comment]
I figured out a workaround in the meantime. Just let the PDF open in Firefox and then add the PDF to Zotero. The metadata is correctly loaded this way.
